extensions/integration-tests/src/test/utils.ts

```diff
@@ -180,6 +180,27 @@ export async function runQuery(query: string, ownerUri: string): Promise<azdata.
 
 }
 
+export async function retryFunction(fn: () => Promise<any>, retryCount: number): Promise<any> {
```

Comment:
Would be better to generic-ify this

`export async function retryFunction<T>(fun: () => Promise<T>, retryCount: number): Promise<T> {`

Reply:
@chgagnon@microsoft.com please let me know if you have more comments.

---
In reply to: [535489452](https://github.com/microsoft/azuredatastudio/pull/13649#discussion_r535489452) [](ancestors = 535489452)